scripts/linters/general_purpose_linter_test.py

```diff
@@ -636,6 +638,17 @@ def test_invalid_use_of_iterkeys(self):
             ['Line 43: Please use keys() instead.'],
             self.linter_stdout)
 
+    def test_invalid_use_of_assert_raises(self):
+        with self.print_swap:
+            general_purpose_linter.GeneralPurposeLinter(
+                [INVALID_ASSERT_RAISES_FILEPATH], FILE_CACHE, True
+            ).perform_all_lint_checks()
+        self.assert_same_list_elements([
+            'Line 40: Please use assertRaises method instead use '
```

Comment:
This docstring needs to be clearer.

Please do not use the assertRaises method. Instead, use assertRaisesRegexp and provide a sufficiently strong regexp string to validate that the correct error is being raised.